blueocean-rest-impl/src/main/resources/io/jenkins/blueocean/service/embedded/Messages.properties

```diff
@@ -1 +1,2 @@
-BlueOceanUrlAction.DisplayName=Open Blue Ocean
\ No newline at end of file
+BlueOceanUrlAction.DisplayName=Open Blue Ocean
+BlueOceanUrlAction.GenericDisplayName=Open Blue Ocean Homepage
```

Comment:
really not useful 

build url:
https://build.platform.hmcts.net/view/<view name>/job/org-folder/job/repo-n/job/name/master/541/console

![image](https://github.com/jenkinsci/blueocean-plugin/assets/21194782/efcbaf4f-c79b-4b6b-b1ba-0794d6ffa405)

It opens a run page just fine.

This is really too long for a side panel link

Reply:
Agree. This need to be fixed. I will revert this change

Reply:
https://github.com/jenkinsci/blueocean-plugin/pull/2474

Reply:
This was added because every day I get frustrated that the "Blue Ocean" link takes me to a useless homepage that just shows the list of PRs when I click the link from my PR page - it only goes to a useful page when I select it from an actual build.

Reply:
@timja @olamy

Reply:
it would be better to fix that instead if possible then?

Blue ocean doesn't have an equivalent page for that view though which is why.

It could go to either:
https://ci.jenkins.io/blue/organizations/jenkins/Core%2Fjenkins/pr
or
Just the last build on the PR

----

see this link here:
https://ci.jenkins.io/job/Core/job/jenkins/view/change-requests/job/PR-5597/134/

the change made here added the 'homepage' link to that view which is wrong because that opens a correct page.

----

Blue Ocean Dashboard would probably be okay as a term for just the case where there's no matching blue ocean page I think, I guess this doesn't affect many people as they will normally open their build from a link on their SCM provider which will have a direct link to the right place so I would say it's a bit unusual to open it from there.